examples/paho-mqtt/main.c

```diff
@@ -293,6 +296,7 @@ static unsigned char readbuf[BUF_SIZE];
 
 int main(void)
 {
+    msg_init_queue(_main_msg_queue, MAIN_QUEUE_SIZE);
```

Comment:
Why is this needed with GNRC?

Reply:
For ping (if that is used), as the executor of the `ping` command is the main thread.

Reply:
My students just learned this the hard way. ;-)

Reply:
You could put a 

```suggestion
    if (IS_USED(MODULE_GNRC_ICMPV6_ECHO)) {
        msg_init_queue(_main_msg_queue, MAIN_QUEUE_SIZE);
    }
```

here then